sentry.client.config.js

```diff
@@ -5,20 +5,33 @@
 import { WebError } from "@fwl/web/dist/errors";
 import * as Sentry from "@sentry/nextjs";
 
-Sentry.init({
-  dsn: "https://6baf0213093a428d9e9ec7ae0979868c@o287463.ingest.sentry.io/5387583",
-  tracesSampleRate: 1.0,
-  environment:
-    process.env.NEXT_PUBLIC_SENTRY_ENVIRONMENT || process.env.NODE_ENV,
-  enabled:
-    process.env.NEXT_PUBLIC_SENTRY_ENVIRONMENT === "development"
-      ? false
-      : process.env.NODE_ENV === "production",
-  beforeSend(event, hint) {
-    if (hint.originalException instanceof WebError) {
-      return null;
-    }
+const sentryUserConsent = document.cookie
```

Comment:
can we surround all this with a try/catch? 
It would prevent a crash in cases like `startsWith` is not defined, or a JSON.parse error or a zealous adblocker that makes the `Sentry` object empty

Reply:
You mean all the entire script right ?

Reply:
yes, sorry, I should have made a multiline comment but I was lazy 🙇‍♂️

Reply:
Ahah no worries, I suspected it but it's better to be sure 😛